.github/workflows/ci-workflow.yml

```diff
@@ -113,6 +113,12 @@ jobs:
     name: Functional tests with Ragger (compatible with Speculos & physical devices)
     needs:
       - nano_build
+    strategy:
```

Comment:
This is sub-optimal as this job has to wait for **all** the builds to be done (and downloads all the binaries). However there is no ideal solution. Re-integrating this job into the `nano_build` one would allow functional tests to start as soon as the related build is finished, however we would lose some detail on the CI (there will not be a `functional_test` job anymore, only bigger `build and test` with no immediate feedback on what is failing in case of error.

Reply:
I agree that we are waiting to much, but I think that this is still the best solution. what do you think about it.

Reply:
I don't have strong opinion on this, both solutions are equally as good as GitHub allows.